### ISSUE TITLE
Problem: description is no longer added to collection

### ISSUE THREAD

**Issue by gazhayes** [EmurgoHK/Emurgis#46]
Problem: issue descriptions are no longer added to the mongo collection. This was working before.

**Comment by AshHimself** [comment]
@samuelralak - I think this may be related to your PR as well mate..

```
+		Problems.insert({
-        // if (Meteor.userId()) {	+			'summary': summary,
-        Problems.insert({	+			'description': description || "",
-            'summary': summary,	+			'description': solution || "",
-            'description': description || "",	+			'createdAt': new Date().getTime(),
-            'createdAt': new Date().getTime(),	+			'createdBy': Meteor.userId() || ""
-            'createdBy': Meteor.userId() || ""	+		})
```

**Comment by samuelralak** [comment]
right. claiming this. 

@AshHimself from which PR were your changes deleted?

**Comment by AshHimself** [comment]
https://github.com/EmurgoHK/Emurgis/pull/41

**Comment by samuelralak** [comment]
thanks

**Comment by samuelralak** [comment]
fixed in [#48](https://github.com/EmurgoHK/Emurgis/pull/48)
